go.mod

```diff
@@ -284,10 +284,6 @@ require (
 // Using a fork of Prometheus with Mimir-specific changes.
 replace github.com/prometheus/prometheus => github.com/grafana/mimir-prometheus v0.0.0-20241003114040-35ec40c3b27a
 
-// client_golang v1.20.3 has some data races in histogram exemplars.
-// Stick to v1.19.1 until they are fixed.
-replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.19.1
```

Comment:
@krajorama are these races now fixed in v1.20.4? Could look like it from the [release notes](https://github.com/prometheus/client_golang/releases/tag/v1.20.4).

Reply:
should be

Reply:
I hear from @krajorama in chat that this problem _should_ be fixed.